src/main/java/uk/gov/hmcts/reform/sendletter/exceptions/LetterAlreadySentException.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.reform.sendletter.exceptions;
+
+public class LetterAlreadySentException extends RuntimeException {
```

Comment:
Do we want to add some additional constructors as below.
```
 public LetterAlreadySentException() {
          super();
      }
      public LetterAlreadySentException(String s) {
          super(s);
      }

```

Reply:
no need until we want different messages for different scenarios imho

Reply:
👍